src/librustc/infer/error_reporting/mod.rs

```diff
@@ -303,28 +303,28 @@ impl<'a, 'gcx, 'tcx> InferCtxt<'a, 'gcx, 'tcx> {
     ) {
         debug!("report_region_errors(): {} errors to start", errors.len());
 
-        if will_later_be_reported_by_nll && self.tcx.nll() {
+        if will_later_be_reported_by_nll && self.tcx.use_mir() {
```

Comment:
eek; while I understand collapsing `-Z nll` into `-Z borrowck=mir`, you've chosen the method name "use_mir" to represent this collapsed flag? I would have expected you to either reuse "nll" again, or to include something about borrow-checking in the method name... "use_mir" just seems very broad and easy to misinterpret...

Reply:
indeed I should rename it =)